sherpa/plot/__init__.py

```diff
@@ -73,15 +73,19 @@
 _ = numpy.seterr(invalid='ignore')
 
 plot_opt = config.get('options', 'plot_pkg', fallback='dummy')
-plot_opt = [o.strip().lower() + '_backend' for o in plot_opt.split()]
+plot_opt = [o.strip().lower() for o in plot_opt.split()]
+
+plot_backends = {'pylab': 'sherpa.plot.pylab_backend.PylabBackend',
+                 'matplotlib': 'sherpa.plot.pylab_backend.PylabBackend',
+                 'dummy': 'sherpa.plot.backends.BasicBackend',
+                 }
 
 backend = None
 '''Currently active backend module for plotting.'''
 
 for plottry in plot_opt:
     try:
-        backend = importlib.import_module('.' + plottry,
-                                          package='sherpa.plot')
+        backend_class = importlib.import_module()
```

Comment:
I am hoping this is changed in a later commit otherwise I don't understand. anything.